driver/main.ml

```diff
@@ -142,6 +142,12 @@ module Options = Main_args.Make_bytecomp_options (struct
   let anonymous = anonymous
 end)
 
+let vmthread_deprecated_message = "\
+The -vmthread argument of ocamlc is deprecated\n\
+and will soon be abandoned. For VM-level scheduling, consider using a\n\
+third-party library such as Lwt. Otherwise, please switch to system\n\
```

Comment:
I don't find the mention of `Lwt` particularly enlighting here since IIRC vm-thread actually implemented a compatibility layer for the system threads library.

Reply:
I think it is useful to mention LWT, but perhaps in position number 2?  E.g.

The -vmthread argument of ocamlc is deprecated\n\
and will soon be abandoned. Please switch to system threads,\n\
which have the same API.  Lightweight threads with VM-level scheduling\n\
are provided by third-party libraries such as Lwt, but with a different API.